runtime/doc/intro.txt

```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *bram*
+
+NeoVim is a text editor built as a continuation of Vim, Vi Improved. Vim was
+orginally developed by Bram Moolenaar.  Searching his name within the source
+code of NeoVim will reveal just how much of his work still remains in
+NeoVim.  On August 3, 2023, he passed away at the age of 62.  If Vim or NeoVim
```

Comment:
```suggestion
Nvim is a text editor built as a continuation of Vim, Vi Improved. Vim was
orginally developed by Bram Moolenaar.  Searching his name within the source
code of Nvim will reveal just how much of his work still remains in
Nvim.  On August 3, 2023, he passed away at the age of 62.  If Vim or Nvim
```
The text editor is called "Nvim" (launched through the command `nvim`). "Neovim" is the whole project.